albumentations/imgaug/transforms.py

```diff
@@ -22,13 +25,45 @@ def apply(self, img, **params):
 
 
 class DualIAATransform(DualTransform, BasicIAATransform):
-    pass
+    def __init__(self, p):
+        super(DualIAATransform, self).__init__(p)
+
+    def apply_to_bboxes(self, bboxes, **params):
+        rows, cols = params['rows'], params['cols']
+
+        bboxes = convert_bboxes_from_albumentations((rows, cols), bboxes, 'pascal_voc')
+
+        bboxes_t = ia.BoundingBoxesOnImage([ia.BoundingBox(*bbox[:4]) for bbox in bboxes], (rows, cols))
+        bboxes_t = self.deterministic_processor.augment_bounding_boxes([bboxes_t])[0].bounding_boxes
+        bboxes_t = [[bbox.x1, bbox.y1, bbox.x2, bbox.y2] + list(bbox_orig[4:]) for (bbox, bbox_orig) in
+                    zip(bboxes_t, bboxes)]
+
+        bboxes = convert_bboxes_to_albumentations((rows, cols), bboxes_t, 'pascal_voc')
+        return bboxes
 
 
 class ImageOnlyIAATransform(ImageOnlyTransform, BasicIAATransform):
     pass
 
 
+class IAACropAndPad(DualIAATransform):
+    def __init__(self, px=None, percent=None, pad_mode='constant', pad_cval=0, keep_size=True, p=1):
+        super(IAACropAndPad, self).__init__(p)
+        self.processor = iaa.CropAndPad(px, percent, pad_mode, pad_cval, keep_size)
+
+
+class IAAFliplr(DualIAATransform):
+    def __init__(self, p=0.5):
+        super(IAAFliplr, self).__init__(p)
+        self.processor = iaa.Fliplr(1)
```

Comment:
Shouldn't it be `iaa.Fliplr(p)` because we want to pass the probability of applying `iaa.Fliplr` and this probability could differ from 1?

Reply:
Probably you're right, that we should pass parameter `p` to `iaa.Fliplr`, and initialize `super(IAAFliplr, self).__init__(1)` to unconditional forward calls to imgaug transformations.